app/models/user.py

```diff
@@ -27,16 +27,18 @@ class User(ModelMixin):
     date_created = db.Column(db.DateTime, default=db.func.current_timestamp())
     last_seen = db.Column(db.DateTime, default=db.func.current_timestamp())
     projects = db.relationship('Project', backref='owner', lazy=True)
+    organisation = db.Column(db.String(256), nullable=True, default="")
     other_projects = db.relationship('ProjectUser', back_populates='user')
     is_beta_user = db.Column(db.Boolean, nullable=False, default=False)
     credits = db.relationship('Credit', backref='user', lazy=True)
     credit_assignments = db.relationship('CreditAssignment', backref='user', lazy=True)
 
-    def __init__(self, email, name, password):
+    def __init__(self, email, name, password, organisation):
```

Comment:
@LanternNassi this line makes the tests fail as the organisation field is not reflected/updated in the test cases. So you can resolve this by adding the organization field in the positive test case with an example value See error below 

<img width="883" alt="Screenshot 2023-06-21 at 11 33 26" src="https://github.com/crane-cloud/backend/assets/36065782/7afded43-1f65-43a0-bc5e-0098ce56b3bf">